models/base_model.py

```diff
@@ -27,7 +27,6 @@ def delete(cls, id):
     @classmethod
     def create(cls, schema, payload):
         attrs = cls.validate(schema, payload)
-
         obj = cls(**attrs)
```

Comment:
This is a personal style thing: Perhaps we can add this space back in?
```suggestion

        obj = cls(**attrs)
```